packages/frontend/core/src/mobile/pages/workspace/detail/mobile-detail-page.tsx

```diff
@@ -215,12 +216,12 @@ const notFound = (
   </>
 );
 
-const JournalDetailPage = ({
+const MobileDetailPage = ({
   pageId,
   date,
 }: {
   pageId: string;
-  date: string;
+  date?: string;
 }) => {
   const journalService = useService(JournalService);
   const { openJournal } = useJournalRouteHelper();
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +229]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->I separated Page/Journal mainly for avoiding doing this heavy calculation.<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/8760?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
When journal mode is toggled the whole page will get remounted. I think the calculation should be muted when mode is a normal doc.